EmpowerPlant.xcodeproj/project.pbxproj

```diff
@@ -264,7 +266,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "if which sentry-cli >/dev/null; then\n    if [ -f .env ] && grep -q \"^SENTRY_ORG=\" .env && grep -q \"^SENTRY_PROJECT=\" .env; then\n        export $(grep -v '^#' .env | sed '/^\\s*$/d' | xargs)\n    else\n        echo \"[ERROR] .env does not exist or does not have SENTRY_ORG and SENTRY_PROJECT defined\"\n        exit 1\n    fi\n    if [ ! -n \"$SENTRY_AUTH_TOKEN\" ]; then\n        if [ -f ~/.sentryclirc ]; then\n            export SENTRY_AUTH_TOKEN=$(grep -oE \"token=([^\\n\\r]*)$\" ~/.sentryclirc | cut -d'=' -f2)\n        else \n            echo \".sentryclirc does not exist.\"\n        fi\n        if [ -f ~/.zshrc ] && grep -q \"export SENTRY_AUTH_TOKEN\" ~/.zshrc; then\n            grep -m 1 \"export SENTRY_AUTH_TOKEN\" ~/.zshrc > /tmp/ios.sentry-build.tmp && source /tmp/ios.sentry-build.tmp && rm /tmp/ios.sentry-build.tmp\n        else\n            echo \".zshrc does not exist or does not have SENTRY_AUTH_TOKEN in it.\"\n        fi\n    fi\n    if [ ! -n \"$SENTRY_AUTH_TOKEN\" ]; then\n        echo \"[ERROR] must provide SENTRY_AUTH_TOKEN either through command line, .zshrc or .sentryclirc\"\n        exit 1\n    fi\n\n    ERROR=$(sentry-cli upload-dif --include-sources \"$DWARF_DSYM_FOLDER_PATH\" 2>&1 >/dev/null)\n    if [ ! $? -eq 0 ]; then\n        echo \"warning: sentry-cli - $ERROR\"\n    fi\nelse\n    echo \"[ERROR] sentry-cli not installed, download from https://github.com/getsentry/sentry-cli/releases\"\n    exit 1\nfi\n";
```

Comment:
If this ever changed, it's also much harder to read a diff if it's written here vs in a separate file, like below